src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write code here
+const head = document.querySelector('thead');
+const body = document.querySelectorAll('tbody tr');
+const tbody = document.querySelector('tbody');
+
+function converter(ifSalary) {
```

Comment:
ifSalary это булиан, это не может быть число или строка
```suggestion
function converter(salary) {
```